add-to-org.rb

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env ruby
+# add USERNAME as member to ORGANIZATION and optionally to TEAM
+# assumes you created and stored the appropriate OAuth token as an ENV variable called GITHUBUSER_TOKEN
+
+require 'octokit'
+require 'optparse'
+
+# Make sure arguments are specified
+ARGV << '-h' if ARGV.empty?
```

Comment:
yeah i thought so too. Unfortunately I cant find anything in github/github, github/shell, github/cheatsheets, or anywhere else that include any good examples. And this method is numero uno on stack overflow. :man_shrugging: